*** Starting uWSGI 2.0.23 (64bit) on [Sun Mar 31 07:26:12 2024] ***
compiled with version: 9.4.0 on 07 November 2023 21:42:00
os: Linux-5.4.0-136-generic #153-Ubuntu SMP Thu Nov 24 15:56:58 UTC 2022
nodename: studentlearn1
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /var/www/html/LivingArchiveOfAboriginalLanguages
detected binary path: /var/www/html/LivingArchiveOfAboriginalLanguages/env/bin/uwsgi
setuid() to 8
chdir() to ./
your processes number limit is 31476
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to TCP address 127.0.0.1:8006 fd 3
Python version: 3.11.7 (main, Dec  8 2023, 18:56:57) [GCC 9.4.0]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x7f933581dc98
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 145840 bytes (142 KB) for 1 cores
*** Operational MODE: single process ***
Traceback (most recent call last):
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/core/handlers/wsgi.py", line 125, in __init__
    self.load_middleware()
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/core/handlers/base.py", line 40, in load_middleware
    middleware = import_string(middleware_path)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/utils/module_loading.py", line 30, in import_string
    return cached_import(module_path, class_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/utils/module_loading.py", line 15, in cached_import
    module = import_module(module_path)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/wagtail/contrib/redirects/middleware.py", line 7, in <module>
    from wagtail.contrib.redirects import models
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/wagtail/contrib/redirects/models.py", line 7, in <module>
    from wagtail.models import Page
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/wagtail/models/__init__.py", line 22, in <module>
    from django.contrib.auth.models import Group, Permission
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/contrib/auth/models.py", line 3, in <module>
    from django.contrib.auth.base_user import AbstractBaseUser, BaseUserManager
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/contrib/auth/base_user.py", line 50, in <module>
    class AbstractBaseUser(models.Model):
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/db/models/base.py", line 127, in __new__
    app_config = apps.get_containing_app_config(module)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/apps/registry.py", line 260, in get_containing_app_config
    self.check_apps_ready()
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/apps/registry.py", line 138, in check_apps_ready
    raise AppRegistryNotReady("Apps aren't loaded yet.")
django.core.exceptions.AppRegistryNotReady: Apps aren't loaded yet.
unable to load app 0 (mountpoint='') (callable not found or import error)
Traceback (most recent call last):
  File "/usr/lib/python3.11/logging/config.py", line 573, in configure
    handler = self.configure_handler(handlers[name])
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/logging/config.py", line 757, in configure_handler
    result = factory(**kwargs)
             ^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/logging/__init__.py", line 1181, in __init__
    StreamHandler.__init__(self, self._open())
                                 ^^^^^^^^^^^^
  File "/usr/lib/python3.11/logging/__init__.py", line 1213, in _open
    return open_func(self.baseFilename, self.mode,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
PermissionError: [Errno 13] Permission denied: '/var/www/html/LivingArchiveOfAboriginalLanguages/debug.log'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "./livingarchive/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
                  ^^^^^^^^^^^^^^^^^^^^^^
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/core/wsgi.py", line 12, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/__init__.py", line 19, in setup
    configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/utils/log.py", line 76, in configure_logging
    logging_config_func(logging_settings)
  File "/usr/lib/python3.11/logging/config.py", line 823, in dictConfig
    dictConfigClass(config).configure()
  File "/usr/lib/python3.11/logging/config.py", line 580, in configure
    raise ValueError('Unable to configure handler '
ValueError: Unable to configure handler 'file'
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 3551267)
spawned uWSGI worker 1 (pid: 3551268, cores: 1)
SIGINT/SIGTERM received...killing workers...
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.23 (64bit) on [Sun Mar 31 07:26:42 2024] ***
compiled with version: 9.4.0 on 07 November 2023 21:42:00
os: Linux-5.4.0-136-generic #153-Ubuntu SMP Thu Nov 24 15:56:58 UTC 2022
nodename: studentlearn1
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /var/www/html/LivingArchiveOfAboriginalLanguages
detected binary path: /var/www/html/LivingArchiveOfAboriginalLanguages/env/bin/uwsgi
setuid() to 8
chdir() to ./
your processes number limit is 31476
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to TCP address 127.0.0.1:8002 fd 3
Python version: 3.11.7 (main, Dec  8 2023, 18:56:57) [GCC 9.4.0]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x7f20857a6c98
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 145840 bytes (142 KB) for 1 cores
*** Operational MODE: single process ***
Traceback (most recent call last):
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/core/handlers/wsgi.py", line 125, in __init__
    self.load_middleware()
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/core/handlers/base.py", line 40, in load_middleware
    middleware = import_string(middleware_path)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/utils/module_loading.py", line 30, in import_string
    return cached_import(module_path, class_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/utils/module_loading.py", line 15, in cached_import
    module = import_module(module_path)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/wagtail/contrib/redirects/middleware.py", line 7, in <module>
    from wagtail.contrib.redirects import models
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/wagtail/contrib/redirects/models.py", line 7, in <module>
    from wagtail.models import Page
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/wagtail/models/__init__.py", line 22, in <module>
    from django.contrib.auth.models import Group, Permission
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/contrib/auth/models.py", line 3, in <module>
    from django.contrib.auth.base_user import AbstractBaseUser, BaseUserManager
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/contrib/auth/base_user.py", line 50, in <module>
    class AbstractBaseUser(models.Model):
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/db/models/base.py", line 127, in __new__
    app_config = apps.get_containing_app_config(module)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/apps/registry.py", line 260, in get_containing_app_config
    self.check_apps_ready()
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/apps/registry.py", line 138, in check_apps_ready
    raise AppRegistryNotReady("Apps aren't loaded yet.")
django.core.exceptions.AppRegistryNotReady: Apps aren't loaded yet.
unable to load app 0 (mountpoint='') (callable not found or import error)
Traceback (most recent call last):
  File "/usr/lib/python3.11/logging/config.py", line 573, in configure
    handler = self.configure_handler(handlers[name])
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/logging/config.py", line 757, in configure_handler
    result = factory(**kwargs)
             ^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/logging/__init__.py", line 1181, in __init__
    StreamHandler.__init__(self, self._open())
                                 ^^^^^^^^^^^^
  File "/usr/lib/python3.11/logging/__init__.py", line 1213, in _open
    return open_func(self.baseFilename, self.mode,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
PermissionError: [Errno 13] Permission denied: '/var/www/html/LivingArchiveOfAboriginalLanguages/debug.log'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "./livingarchive/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
                  ^^^^^^^^^^^^^^^^^^^^^^
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/core/wsgi.py", line 12, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/__init__.py", line 19, in setup
    configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/utils/log.py", line 76, in configure_logging
    logging_config_func(logging_settings)
  File "/usr/lib/python3.11/logging/config.py", line 823, in dictConfig
    dictConfigClass(config).configure()
  File "/usr/lib/python3.11/logging/config.py", line 580, in configure
    raise ValueError('Unable to configure handler '
ValueError: Unable to configure handler 'file'
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 3551269)
spawned uWSGI worker 1 (pid: 3551270, cores: 1)
--- no python application found, check your startup logs for errors ---
[pid: 3551270|app: -1|req: -1/1] 1.121.134.198 () {58 vars in 1053 bytes} [Sun Mar 31 07:26:45 2024] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
SIGINT/SIGTERM received...killing workers...
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.23 (64bit) on [Sun Mar 31 07:28:07 2024] ***
compiled with version: 9.4.0 on 07 November 2023 21:42:00
os: Linux-5.4.0-136-generic #153-Ubuntu SMP Thu Nov 24 15:56:58 UTC 2022
nodename: studentlearn1
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /var/www/html/LivingArchiveOfAboriginalLanguages
detected binary path: /var/www/html/LivingArchiveOfAboriginalLanguages/env/bin/uwsgi
setuid() to 8
chdir() to ./
your processes number limit is 31476
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to TCP address 127.0.0.1:8002 fd 3
Python version: 3.11.7 (main, Dec  8 2023, 18:56:57) [GCC 9.4.0]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x7f761d551c98
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 145840 bytes (142 KB) for 1 cores
*** Operational MODE: single process ***
Traceback (most recent call last):
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/core/handlers/wsgi.py", line 125, in __init__
    self.load_middleware()
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/core/handlers/base.py", line 40, in load_middleware
    middleware = import_string(middleware_path)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/utils/module_loading.py", line 30, in import_string
    return cached_import(module_path, class_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/utils/module_loading.py", line 15, in cached_import
    module = import_module(module_path)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/wagtail/contrib/redirects/middleware.py", line 7, in <module>
    from wagtail.contrib.redirects import models
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/wagtail/contrib/redirects/models.py", line 7, in <module>
    from wagtail.models import Page
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/wagtail/models/__init__.py", line 22, in <module>
    from django.contrib.auth.models import Group, Permission
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/contrib/auth/models.py", line 3, in <module>
    from django.contrib.auth.base_user import AbstractBaseUser, BaseUserManager
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/contrib/auth/base_user.py", line 50, in <module>
    class AbstractBaseUser(models.Model):
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/db/models/base.py", line 127, in __new__
    app_config = apps.get_containing_app_config(module)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/apps/registry.py", line 260, in get_containing_app_config
    self.check_apps_ready()
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/apps/registry.py", line 138, in check_apps_ready
    raise AppRegistryNotReady("Apps aren't loaded yet.")
django.core.exceptions.AppRegistryNotReady: Apps aren't loaded yet.
unable to load app 0 (mountpoint='') (callable not found or import error)
Traceback (most recent call last):
  File "/usr/lib/python3.11/logging/config.py", line 573, in configure
    handler = self.configure_handler(handlers[name])
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/logging/config.py", line 757, in configure_handler
    result = factory(**kwargs)
             ^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/logging/__init__.py", line 1181, in __init__
    StreamHandler.__init__(self, self._open())
                                 ^^^^^^^^^^^^
  File "/usr/lib/python3.11/logging/__init__.py", line 1213, in _open
    return open_func(self.baseFilename, self.mode,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
PermissionError: [Errno 13] Permission denied: '/var/www/html/LivingArchiveOfAboriginalLanguages/debug.log'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "./livingarchive/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
                  ^^^^^^^^^^^^^^^^^^^^^^
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/core/wsgi.py", line 12, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/__init__.py", line 19, in setup
    configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/utils/log.py", line 76, in configure_logging
    logging_config_func(logging_settings)
  File "/usr/lib/python3.11/logging/config.py", line 823, in dictConfig
    dictConfigClass(config).configure()
  File "/usr/lib/python3.11/logging/config.py", line 580, in configure
    raise ValueError('Unable to configure handler '
ValueError: Unable to configure handler 'file'
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 3551289)
spawned uWSGI worker 1 (pid: 3551290, cores: 1)
--- no python application found, check your startup logs for errors ---
[pid: 3551290|app: -1|req: -1/1] 65.155.30.101 () {52 vars in 887 bytes} [Sun Mar 31 07:28:09 2024] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 3551290|app: -1|req: -1/2] 1.121.134.198 () {58 vars in 1053 bytes} [Sun Mar 31 07:28:10 2024] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 3551290|app: -1|req: -1/3] 1.121.134.198 () {58 vars in 1053 bytes} [Sun Mar 31 07:28:12 2024] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 3551290|app: -1|req: -1/4] 1.121.134.198 () {58 vars in 1053 bytes} [Sun Mar 31 07:28:12 2024] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
SIGINT/SIGTERM received...killing workers...
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.23 (64bit) on [Sun Mar 31 07:29:07 2024] ***
compiled with version: 9.4.0 on 07 November 2023 21:42:00
os: Linux-5.4.0-136-generic #153-Ubuntu SMP Thu Nov 24 15:56:58 UTC 2022
nodename: studentlearn1
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /var/www/html/LivingArchiveOfAboriginalLanguages
detected binary path: /var/www/html/LivingArchiveOfAboriginalLanguages/env/bin/uwsgi
setuid() to 8
chdir() to ./
your processes number limit is 31476
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to TCP address 127.0.0.1:8002 fd 3
Python version: 3.11.7 (main, Dec  8 2023, 18:56:57) [GCC 9.4.0]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x7fe68da7ec98
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 145840 bytes (142 KB) for 1 cores
*** Operational MODE: single process ***
Traceback (most recent call last):
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/core/handlers/wsgi.py", line 125, in __init__
    self.load_middleware()
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/core/handlers/base.py", line 40, in load_middleware
    middleware = import_string(middleware_path)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/utils/module_loading.py", line 30, in import_string
    return cached_import(module_path, class_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/utils/module_loading.py", line 15, in cached_import
    module = import_module(module_path)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/wagtail/contrib/redirects/middleware.py", line 7, in <module>
    from wagtail.contrib.redirects import models
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/wagtail/contrib/redirects/models.py", line 7, in <module>
    from wagtail.models import Page
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/wagtail/models/__init__.py", line 22, in <module>
    from django.contrib.auth.models import Group, Permission
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/contrib/auth/models.py", line 3, in <module>
    from django.contrib.auth.base_user import AbstractBaseUser, BaseUserManager
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/contrib/auth/base_user.py", line 50, in <module>
    class AbstractBaseUser(models.Model):
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/db/models/base.py", line 127, in __new__
    app_config = apps.get_containing_app_config(module)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/apps/registry.py", line 260, in get_containing_app_config
    self.check_apps_ready()
  File "/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/django/apps/registry.py", line 138, in check_apps_ready
    raise AppRegistryNotReady("Apps aren't loaded yet.")
django.core.exceptions.AppRegistryNotReady: Apps aren't loaded yet.
unable to load app 0 (mountpoint='') (callable not found or import error)
Warning: DEFAULT_PDF_VIEW_PROVIDER unspecified.Make sure you have either django-weasyprint or django-tex installed, or provide a default.
/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/wagtail/search/index.py:301: RemovedInWagtail60Warning: The partial_match option on SearchField has no effect and will be removed. Use AutocompleteField instead
  warn(
/var/www/html/LivingArchiveOfAboriginalLanguages/env/lib/python3.11/site-packages/wagtail/contrib/modeladmin/apps.py:15: RemovedInWagtail60Warning: wagtail.contrib.modeladmin is deprecated. Use wagtail.snippets or the separate wagtail-modeladmin package instead.
  warn(
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x7fe68da7ec98 pid: 3551300 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 3551300)
spawned uWSGI worker 1 (pid: 3551301, cores: 1)
[pid: 3551301|app: 0|req: 1/1] 1.121.134.198 () {58 vars in 1053 bytes} [Sun Mar 31 07:29:09 2024] GET / => generated 26349 bytes in 729 msecs (HTTP/1.1 200) 8 headers in 365 bytes (1 switches on core 0)
SIGINT/SIGTERM received...killing workers...
worker 1 buried after 1 seconds
goodbye to uWSGI.
